README.md

```diff
@@ -167,18 +181,12 @@ indexed metadata.
     }'
 
 #### CI/CD with Travis CI
-We use [Travis CI](https://travis-ci.org/HumanCellAtlas/data-store) for continuous integration testing and
-deployment. When `make test` succeeds, Travis CI deploys the application into the `dev` stage on AWS for every commit
-that goes on the master branch. This behavior is defined in the `deploy` section of `.travis.yml`.
+
+We use [Travis CI](https://travis-ci.org/HumanCellAtlas/data-store) for continuous integration testing and deployment. When `make test` succeeds, Travis CI deploys the application into the `dev` stage on AWS for every commit that goes on the master branch. This behavior is defined in the `deploy` section of `.travis.yml`.
 
 #### Authorizing Travis CI to deploy
-Encrypted environment variables give Travis CI the AWS credentials needed to run the tests and deploy the app. Run
-`scripts/authorize_aws_deploy.sh IAM-PRINCIPAL-TYPE IAM-PRINCIPAL-NAME` (e.g. `authorize_aws_deploy.sh user hca-test`)
-to give that principal the permissions needed to deploy the app. Because this is a limited set of permissions, it does
-not have write access to IAM. To set up the IAM policies for resources in your account that the app will use, run `make
-deploy` using privileged account credentials once from your workstation. After this is done, Travis CI will be able to
-deploy on its own. You must repeat the `make deploy` step from a privileged account any time you change the IAM policies
-in `policy.json.template` files.
+
```

Comment:
any time you change the IAM policies templates in `iam/policy-templates/`

Reply:
Fixed.